TLM/TLM/Manager/Impl/OptionsManager.cs

```diff
@@ -18,13 +18,31 @@ public class OptionsManager
             Log.NotImpl("InternalPrintDebugInfo for OptionsManager");
         }
 
+        /// <summary>
+        /// Converts value to SimulationAccuracy
+        /// </summary>
+        /// <param name="value">Old value</param>
+        /// <returns>SimulationAccuracy value</returns>
+        private static SimulationAccuracy ConvertToSimulationAccuracy(byte value) {
+            return (SimulationAccuracy)Math.Abs(value - 4);
+        }
+
+        /// <summary>
+        /// Converts SimulationAccuracy to SimulationAccuracy
+        /// </summary>
+        /// <param name="value">SimulationAccuracy value</param>
+        /// <returns>byte representation of value (backward compatible)</returns>
+        private static byte ConvertFromSimulationAccuracy(SimulationAccuracy value) {
```

Comment:
Very nice: having a enum instead of int or some float. We should use more of that.